src/backend/web/routes/feeds.js

```diff
@@ -51,4 +51,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', async (req, res) => {
+  const feedData = req.body;
```

Comment:
We need to validate this data, since we require that an `author` and `url` get passed.  If either is missing, you need to send back an HTTP 400, indicating that the request is invalid, and that it's missing data.

Reply:
Just wondering though - if it's a protected route, we will have the author correct? 

So, do I just need to check that there is a URL?

Reply:
We'll have a username, but not necessarily a real, fullname.  Consider the comment you left in Julia's design issue:

- "humphd"
- vs. "David Humphrey"

The author isn't a user, but a display name.  Later, we might consolidate this somehow, but we have a ton of feeds that don't have associated users, so we can't currently rely on that mapping.